core/src/main/java/com/googlecode/psiprobe/AbstractTomcatContainer.java

```diff
@@ -58,7 +58,7 @@
 public abstract class AbstractTomcatContainer implements TomcatContainer {
 
   /** The logger. */
-  protected Log logger = LogFactory.getLog(getClass());
+  protected Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
I think we should reference the actual class name here instead of using getClass().  Same goes for all classes with a logger.  It's faster that way.